examples/react-app/README.md

```diff
@@ -34,6 +34,60 @@ $ architect deploy --local examples/react-app:latest -i app:app
 
 Once the deploy has completed, you can reach your new service by going to http://app.arc.localhost/.
 
+## Creating remote infrastructure
+
+Terraform templates for running the component in a remote environment can be found in the `terraform` directory. Ensure that [`terraform`](https://learn.hashicorp.com/tutorials/terraform/install-cli) is installed and that you have an AWS account, then follow the next steps.
+
+First, navigate to the `terraform` folder in your terminal, then set the environment variables for your AWS credentials like so:
+
+```sh
+export AWS_ACCESS_KEY_ID=<aws_access_key>
+export AWS_SECRET_ACCESS_KEY=<aws_secret_access_key>
+export AWS_DEFAULT_REGION=<aws_region>
+```
+
+Next, create an AWS S3 bucket where the terraform remote state should be stored. Use it in the following command to initialize terraform:
+
+```sh
+terraform init -backend-config="bucket=<s3_bucket_name>" -backend-config="region=<aws_region>" -backend-config="key=<aws_access_key>"
```

Comment:
Can we make a new readme in the terraform folder and move the content there? We can link to it from the main readme.